### PR TITLE
[script.module.protobuf] v2.6.1

### PR DESCRIPTION
**Pull request by WoLpH**
The update for the protobuf2 addon as discussed in #263

### CODE REVIEW

File: script.module.protobuf/addon.xml [hunk 1]
```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.protobuf"
-       name="protobuf"
-       version="2.5.0"
-       provider-name="Valentin-Costel Hăloiu">
-  <requires>
-    <import addon="xbmc.python" version="2.1.0"/>
```
Comment (ronie) [inline script.module.protobuf/addon.xml]: please don't remove the xbmc.python requirement :-)

**Comment by WoLpH** [comment]
Ah, forgot to add that in the new version. It's there now :)

**Comment by Razzeee** [comment]
@ronie 
should be good to go now, shouldn't it?

**Comment by ronie** [comment]
eh....

**Comment by WoLpH** [comment]
This one is fully compatible with the previous one and includes the missing file bug. And now it's at least supported :)

**Comment by ronie** [comment]
thx!

**Pull request merged**